cronograma/r_dev_day/RDevDay-pt.qmd

```diff
@@ -0,0 +1,99 @@
+---
+title: "R Dev Day"
+knitr:
+  opts_chunk:
+    echo: false
+    message: false
+    warning: false
+editor: 
+  markdown: 
+    wrap: sentence
+---
+
+# R Dev Day \@ LatinR 2024
+
+**Datas:** 18 e 19 de novembro de 2024, das 17h30 às 20h (UTC-3)
+
+**Modo:** Online
+
+**Inscrição:** <https://pretix.eu/r-contributors/r-dev-day-latinr2024/>
+
+**Idiomas:** Inglês, português e espanhol
+
+# Sobre
+
+O R Dev Day \@ LatinR 2024 é um evento colaborativo no qual os participantes trabalharão em pequenos grupos para contribuir para o R básico ou para a infraestrutura que dá suporte a essas contribuições, por exemplo
+
+-   Traduzir strings de mensagens para espanhol/português brasileiro
+-   Propor melhorias nos arquivos de ajuda do R
+-   Analisar um erro relatado no R
+-   Desenvolver uma correção para um erro relatado no R
+-   Adicionar conteúdo ao [R Dev Guide](https://contributor.r-project.org/rdevguide/)
+-   Melhorar o [R Translations Dashboard](https://contributor.r-project.org/translations-dashboard/)
+
+Tanto colaboradores novos quanto experientes estão convidados a participar!
+
+## Quando
+
+Haverá duas sessões na segunda-feira, 18 de novembro, e na terça-feira, 19 de novembro. No mesmo horário todos os dias:
+
+| UTC           | Buenos Aires (UTC-3) | Hermosillo (UTC-7) |
```

Comment:
```suggestion
| UTC           | Buenos Aires e Brasília (UTC-3) | Hermosillo (UTC-7) |
```

Reply:
Copy over to English and Spanish translation too